defaults.ini

```diff
@@ -31,6 +31,7 @@ use_system_font=
 readability=
 share=gtfprde
 wallabag=
+wallabag2=
```

Comment:
Since only one of the wallabag keys will be used, could you change `wallabag2` to `wallabag_version`?